_data/sites.json

```diff
@@ -6893,6 +6903,17 @@
         ]
     },
 
+    {
+        "name": "Journal du geek",
+        "url": "http://www.journaldugeek.com/editer-profil/",
+        "difficulty": "easy",
+        "notes": "Log in. Go to your profil, scroll down and click 'Supprimer mon profil'",
+        "notes_fr": "Loggez vous, cliquez sur 'Modifiez mon profil', descendez en bas et cliquez sur 'Supprimer mon profil'",
```

Comment:
Just to confirm, are you fluent in French? We ask people to only translate languages they can speak

Reply:
Fluent, I'm not. But I can read something in French.

Reply:
Ok